webapp/channels/src/components/channel_info_rhs/about_area_channel.tsx

```diff
@@ -27,6 +27,18 @@ const ChannelPurpose = styled.div`
     }
 `;
 
+const ChannelDescriptionHeading = styled.div`
+    color: var(--center-channel-color-75, rgba(63, 67, 80, 0.75));
+    font-family: "Open Sans";
```

Comment:
do we need to specifically add this style, arent font inherited?

Reply:
Yup, removed it.